staging/src/k8s.io/apiextensions-apiserver/pkg/controller/nonstructuralschema/nonstructuralschema_controller.go

```diff
@@ -112,7 +112,7 @@ func calculateCondition(in *apiextensions.CustomResourceDefinition) *apiextensio
 			return cond
 		}
 
-		pth := field.NewPath("spec", "version").Key(v.Name).Child("schema", "openAPIV3Schema")
+		pth := field.NewPath("spec", "versions").Key(v.Name).Child("schema", "openAPIV3Schema")
```

Comment:
should be `NewPath("spec", "versions").Index(i)`

Reply:
oh right